.github/workflows/long-running-azure.yaml

```diff
@@ -41,9 +41,9 @@ on:
   schedule:
     # Run every 2 hours
     - cron: "0 */2 * * *"
-  pull_request:
-    branches:
-      - main
+  # pull_request:
```

Comment:
I disabled this on PR because it takes > an hour. It will still run as scheduled.

Reply:
This only runs the there are changes to the actual workflow file. Please create an issue to re-enable this.

Reply:
https://github.com/radius-project/radius/issues/6497